mmv1/templates/terraform/examples/document_ai_warehouse_location.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_document_ai_warehouse_location" "<%= ctx[:primary_resource_id] %>" {
+    location = "us"
+    project = data.google_project.project.number
+    access_control_mode = "ACL_MODE_DOCUMENT_LEVEL_ACCESS_CONTROL_GCI"
+	database_type = "DB_INFRA_SPANNER"
+}
```

Comment:
Also, is it possible to add these two fields?

Untested fields: document_creator_default_role, kms_key